.docker/Dockerfile.alpine

```diff
@@ -0,0 +1,36 @@
+FROM node:12.22.8-alpine3.14
+
+RUN apk add --no-cache python3 make g++ libc6-compat ttf-dejavu
+
+ADD . /app
+
+MAINTAINER buildmaster@rocket.chat
+
+RUN set -x \
+ && cd /app/bundle/programs/server \
+ && npm install \
+ # Start hack for sharp...
+ && rm -rf npm/node_modules/sharp \
+ && npm install sharp@0.29.3 \
+ && mv node_modules/sharp npm/node_modules/sharp \
+ # End hack for sharp
```

Comment:
```suggestion
 # Start hack for sharp...
 && npm install sharp@$(node -pe 'require("./npm/node_modules/sharp/package.json").version') \
 && mv node_modules/sharp npm/node_modules/sharp \
 # End hack for sharp
```

Reply:
this won't be effective anymore though if sharp doesn't get bundled anymore as @sampaiodiego said -

> I would if we should publish the bundle/tarball without `sharp`

Reply:
yep.. I realised my phrase was misspelled 😂  what I meant was that really would like to not ship sharp with the tarball, so this hack would not be required.. but the on current scenario this hack is required :(